container_service_extension/config_validator.py

```diff
@@ -115,7 +120,8 @@ def get_validated_config(config_file_name,
         nsxt_wire_logger = SERVER_NSXT_WIRE_LOGGER
     check_keys_and_value_types(config, sample_config, location='config file',
                                msg_update_callback=msg_update_callback)
-    _validate_amqp_config(config['amqp'], msg_update_callback)
+    if not use_mqtt:
+        _validate_amqp_config(config['amqp'], msg_update_callback)
```

Comment:
From what I understand, we do not need to validate the mqtt config components because the only field we currently have for the mqtt section is a "verify_ssl" flag. In contrast, for amqp, the host, username, and password must be specified in the config file and validated. For the mqtt extension, we register it ourselves and do not currently need a host or other information. (You could look at the install_cse function in container_service_extension/configure_cse.py to see how we install the mqtt extension without additional information from the config.) Thank you for checking!